backend/src/ipfs.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response } from "express";
+import * as dotenv from "dotenv";
+import { getStarknetId } from "./starknet";
+
+dotenv.config();
+
+interface Proposal {
+  text: string;
+  address: string;
+  starknet_id?: string;
+}
+
+const ADDRES_REGEX: RegExp = /^0x0[0-9a-fA-F]{63}$/;
+const MAX_LENGTH: number = 200;
```

Comment:
10k should be fine, no reason to constrain it a lot

Reply:
Okay